apps/modernization-ui/src/apps/search/patient/PatientCriteria/Address.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react';
+import { Address } from './Address';
+import { PatientCriteriaEntry } from '../criteria';
+import { FormProvider, useForm } from 'react-hook-form';
+import { renderHook } from '@testing-library/react-hooks';
+import { ApolloClient, ApolloProvider, createHttpLink, InMemoryCache } from '@apollo/client';
+
+
+const link = createHttpLink();
+
+const cache = new InMemoryCache();
+
+const client = new ApolloClient({
+    link,
+    cache
+})
+
+const { result } = renderHook(() =>
+    useForm<PatientCriteriaEntry>({
+        mode: 'onChange',
+        defaultValues: { status: [{ name: 'Active', label: 'Active', value: 'ACTIVE' }] }
+    })
+);
+
+const setup = () => {
+    return render (
+        <ApolloProvider client={client}>
```

Comment:
suggestion:  We've used [MockedProvider](https://github.com/CDCgov/NEDSS-Modernization/blob/b5a18a6d5e07ec924731a8914047cb12962eccf0/apps/modernization-ui/src/apps/patient/profile/investigation/PatientInvestigationsTable.spec.tsx#L32) in other tests